dist/tools/doccheck/check.sh

```diff
@@ -6,17 +6,16 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-RIOTBASE=$(readlink -f "$(dirname $(realpath $0))/../../..")
+RIOTBASE="$(cd $(dirname $0)/../../..; pwd)"
```

Comment:
Would just using `realpath -Le` work on mac ? and even remove the second `realpath`.
It would also resolve symlinks.

Reply:
`realpath` is a GNU tools as well, so not available by default on macOS (or FreeBSD). You're right it would resolve symlinks, but does it matter here and provide benefits over simply taking the resulting path?

Reply:
to me this is the most portable version